src/System.Management.Automation/CoreCLR/CorePsAssemblyLoadContext.cs

```diff
@@ -1,19 +1,14 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@iSazonov, your last commit had 3 failures in `PowerShell-CI-linux`
Error occurred in test script '/home/vsts/work/1/s/test/powershell/engine/Basic/Assembly.LoadNative.Tests.ps1'
```powershell
At /home/vsts/work/1/s/test/powershell/engine/Basic/Assembly.LoadNative.Tests.ps1:64 char:16
+         $PSHOME\$sourceDllName | Should -BeNullOrEmpty
+                ~~~~~~~~~~~~~~~
Unexpected token '\$sourceDllName' in expression or statement.
at <ScriptBlock>, /home/vsts/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/home/vsts/work/1/s/test/powershell/engine/Basic/Assembly.LoadNative.Tests.ps1'
```powershell
At /home/vsts/work/1/s/test/powershell/engine/Basic/Assembly.LoadNative.Tests.ps1:64 char:16
+         $PSHOME\$sourceDllName | Should -BeNullOrEmpty
+                ~~~~~~~~~~~~~~~
Unexpected token '\$sourceDllName' in expression or statement.
at <ScriptBlock>, /home/vsts/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/home/vsts/work/1/s/test/powershell/engine/Basic/Assembly.LoadNative.Tests.ps1'
```powershell
At /home/vsts/work/1/s/test/powershell/engine/Basic/Assembly.LoadNative.Tests.ps1:64 char:16
+         $PSHOME\$sourceDllName | Should -BeNullOrEmpty
+                ~~~~~~~~~~~~~~~
Unexpected token '\$sourceDllName' in expression or statement.
at <ScriptBlock>, /home/vsts/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```